src/components/Table.jsx

```diff
@@ -1,9 +1,16 @@
 import PropTypes from 'prop-types';
 import RBTable from 'react-bootstrap/Table';
 
+import data from './data';
 import SearchBar from './SearchBar';
 
-function Table({ team, searchTerm, onSearchChange }) {
+function Table({
+  team,
+  searchTerm,
+  onSearchChange,
+  minValue,
+  maxValue,
+}) {
   return (
     <div>
       <SearchBar onSearchChange={onSearchChange} />
```

Comment:
I suggest you to extract search from here. Search is another filter and not part of a table, so extract it from here